docs/distributing-an-application/installing-on-kubernetes.md

```diff
@@ -26,12 +26,100 @@ kubectl apply -f https://get.replicated.com/kubernetes.yml
 The Kubernetes cluster must already be provisioned.
 
 ### Volumes
-Replicated requires two persistent volumes on the cluster named `replicated-pv` and `replicated-statsd-pv` with a minimum size of 10GB. 
+Replicated requires two persistent volumes on the cluster named `replicated-pv` and `replicated-statsd-pv` with a minimum size of 10GB. However these persistent volumes are created is fine as long as they are created prior to installing Replicated.
 
-If you are running Replicated on GKE, you can do this with the following commands:
+#### Running on GKE
+
+Create the disks for the persistent volumes:
 
 ```bash
 gcloud compute disks create --size=10GB --zone=<zone> replicated-pv
 gcloud compute disks create --size=10GB --zone=<zone> replicated-statsd-pv
 ```
 
+Create the volumes:
+
+replicated.pv
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+ name: replicated-pv
+spec:
+ capacity:
+   storage: 10Gi
+ accessModes:
+   - ReadWriteOnce
+ gcePersistentDisk:
+   pdName: replicated-pv
+   fsType: ext4
+```
+
+replicated-statsd.pv
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+ name: replicated-statsd-pv
+spec:
+ capacity:
+   storage: 10Gi
+ accessModes:
+   - ReadWriteOnce
+ gcePersistentDisk:
+   pdName: replicated-statsd-pv
+   fsType: ext4
+```
+
+Finally add them by running:
+
+```shell
+kubectl create -f replicated.pv
+kubectl create -f replicated-statsd.pv
+```
+
+#### Running Elsewhere
+
+For other environments start with the YAML found below and update the spec to use the persistent volume option that is best suited to your environment. The provided YAML uses hostPath which is not recommended for production as it will not survive a restart.
```

Comment:
`The provided YAML uses hostPath which is not recommended for production as it will not survive a restart.`
- i dont know if this is true. i guess because you put in tmp dir
- i do know that host path will not tolerate when pod is allocated to another node on cluster